components/scream/src/physics/shoc/atmosphere_macrophysics.cpp

```diff
@@ -35,18 +35,21 @@ void SHOCMacrophysics::set_grids(const std::shared_ptr<const GridsManager> grids
 
   // Layout for 2D (1d horiz X 1d vertical) variable
   FieldLayout scalar2d_layout_col{ {COL}, {m_num_cols} };
-  FieldLayout scalar1d_layout_lev{ {LEV},  {m_num_levs} };
+  FieldLayout scalar2d_layout_lev{ {LEV},  {m_num_levs} };
 
   // Layout for 3D (2d horiz X 1d vertical) variable defined at mid-level and interfaces
   FieldLayout scalar3d_layout_mid { {COL,LEV}, {m_num_cols,m_num_levs} };
   FieldLayout scalar3d_layout_int { {COL,ILEV}, {m_num_cols,m_num_levs+1} };
 
+  // Layout for horiz_wind field
+  FieldLayout horiz_wind_layout { {COL,VAR,LEV}, {m_num_cols,2,m_num_levs} };
```

Comment:
Ok, I may have created more confusion than other by having both `VAR` (for Variable) and `CMP` (for Component) as field tags. In my mind, `VAR` was for variables like the tracers, where each "component" represent a completely different variable, possibly unrelated to the other ones, while `CMP` was for a component of a "physically" vector quantity. So in this case, I would have used `CMP` for  the velocity, rather than `VAR`.

The problem is that a field with layout `(COL,VAR,LEV)` is considered different from a field with layout `(COL,CMP,LEV)`, which causes issues with the atm dag. 

Should I get rid of one between `VAR` and `CMP`? It might be that the benefit of having both is minimal, while the risk of mismatched tags is non trivial. If so, I would remove `VAR` and keep `CMP`, which is more generic/flexible, I think. 

Tagging also @AaronDonahue and @jeff-cohere for their opinion on this.

Reply:
Yes, I would agree that we should only have one "generic" sounding dimension tag and that if we had to choose between VAR and CMP that CMP would be the way to go.  VAR does sound like "variable" which might be confusing.  While CMP, component, is more general.

Reply:
Also agree that one component tag is simpler. Let's just make sure to document how CMP is expected to be used and give some examples like the ones above (tracers, vector/tensor components, etc).

Reply:
Ok, let me open an issue about it, so we don't forget.